Gemfile.lock

```diff
@@ -85,6 +85,7 @@ GEM
     mini_mime (1.0.2)
     mini_portile2 (2.4.0)
     minitest (5.14.1)
+    mysql2 (0.5.3)
```

Comment:
Why hasn't gem file changed but gemfile.lock changed ?

Reply:
![image](https://user-images.githubusercontent.com/68435978/90232645-8f871a00-de4f-11ea-9e41-d7bb13967c5f.png)
![image](https://user-images.githubusercontent.com/68435978/90232662-9746be80-de4f-11ea-9f83-dd87a30f236b.png)
i got some conflict after i pull request to github. so i type this coding to fix the conflict. after that my static page home and help is disappear. I think is this coding make the gemfile.lock changed